src/client/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default class Header extends React.Component{
+    state = {
```

Comment:
if we need state, can we use hooks instead? they are much more modern, classes are being phased out

Reply:
Sure thing. I'll need to look into hooks. I'm not familiar with them yet.

Reply:
I could be wrong but it doesnt look like you need state for now. I  would just use functional components